### PR TITLE
add acceptedResponseStatusCodes to uptime check

### PR DESCRIPTION
**Pull request by DrFaust92**
Close https://github.com/hashicorp/terraform-provider-google/issues/12297

If this PR is for Terraform, I acknowledge that I have:

- [x] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [x] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [x] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [x] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
monitoring: add `accepted_response_status_codes` to `monitoring_uptime_check`
```

### CODE REVIEW

File: mmv1/templates/terraform/examples/uptime_check_config_status_code.tf.erb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  timeout      = "60s"
+
+  http_check {
+    path = "some-path"
+    port = "8010"
+    request_method = "POST"
+    content_type = "URL_ENCODED"
+    body = "Zm9vJTI1M0RiYXI="
+
+    accepted_response_status_codes {
```
Comment (ScottSuarez) [inline mmv1/templates/terraform/examples/uptime_check_config_status_code.tf.erb:12]: could your example have an additional status codes block

**Review by ScottSuarez — CHANGES_REQUESTED** [review]
only one small request. Thanks for the work !

File: mmv1/templates/terraform/examples/uptime_check_config_status_code.tf.erb (hunk 1 above)
Comment (DrFaust92) [reply mmv1/templates/terraform/examples/uptime_check_config_status_code.tf.erb:12]: added another exampl

Comment (ScottSuarez) [reply mmv1/templates/terraform/examples/uptime_check_config_status_code.tf.erb:12]: Sorry what I meant here is that this is an array. So you should be able to provide multiple blocks inline for multiple types of accepted status blocks.

For example 
```hcl
resource "google_monitoring_uptime_check_config" "-" {  
...
    accepted_response_status_codes {
      status_class = "STATUS_CLASS_2XX"
    }
    accepted_response_status_codes {
      status_class = "STATUS_CLASS_3XX"
    }
...
}
```

or even 
```hcl
resource "google_monitoring_uptime_check_config" "-" {  
...
    accepted_response_status_codes {
      status_class = "STATUS_CLASS_2XX"
    }
    accepted_response_status_codes {
            status_value = 301
    }
    accepted_response_status_codes {
            status_value = 302
    }
...
}
```

**Review by ScottSuarez — CHANGES_REQUESTED** [review]
Leaving a more detailed description of the request

File: mmv1/templates/terraform/examples/uptime_check_config_status_code.tf.erb (hunk 1 above)
Comment (DrFaust92) [reply mmv1/templates/terraform/examples/uptime_check_config_status_code.tf.erb:12]: gotcha, makes sense.

Comment (DrFaust92) [reply mmv1/templates/terraform/examples/uptime_check_config_status_code.tf.erb:12]: done + tested

**Review by ScottSuarez — APPROVED** [review]
approved pending tests

**Comment by ScottSuarez** [comment]
Thanks for the work !

**Pull request merged**